### PR TITLE
CSI-1553_ds8k_del_vol_with_snap

### PR DESCRIPTION
**Pull request by igorgonibm**

### CODE REVIEW

File: controller/array_action/array_mediator_ds8k.py [hunk 1]
```diff
@@ -336,7 +336,11 @@ def is_volume_has_snapshots(self, volume_id):
         array_volume_id = get_volume_id_from_scsi_identifier(volume_id)
         try:
             array_volume = self.client.get_volume(array_volume_id)
-            return bool(array_volume.flashcopy)
+            flash_copies = array_volume.flashcopy
+            for fc in flash_copies:
+                if fc.sourcevolume == array_volume_id:
```
Comment (ArbelNathan) [inline controller/array_action/array_mediator_ds8k.py:341]: have checked it? Im not sure the it have source_volume

Comment (igorgonibm) [reply controller/array_action/array_mediator_ds8k.py:341]: yeap. Run the code to test

Comment (ArbelNathan) [reply controller/array_action/array_mediator_ds8k.py:341]: if I recall correctly, you dont get there the object.. only id..

Comment (ArbelNathan) [reply controller/array_action/array_mediator_ds8k.py:341]: oh if its work then ok

**Pull request merged**